regression-test/plugins/plugins_create_table_nested_type.groovy

```diff
@@ -18,75 +18,125 @@
 import org.apache.doris.regression.suite.Suite
 
 // create table with nested data type, now default complex data include array, map, struct
-Suite.metaClass.create_table_with_nested_type = { int maxDepth, String tbName /* param */ ->
+Suite.metaClass.create_table_with_nested_type = { int maxDepth, def typeArr, String tbName /* param */ ->
     Suite suite = delegate as Suite
 
+    if (typeArr.size() != maxDepth) {
+        println("level not equal typeArr")
+        return
```

Comment:
throw exception？

Reply:
Manual calls to the old code would have made errors, and the current code is guaranteed not to make errors here